ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/NewMigrationCommand.java

```diff
@@ -31,7 +35,11 @@
 
   @Override
   public void run() {
-    throw new UnsupportedOperationException();
+    try {
+      Files.createDirectories(Paths.get(projectPath));
+      new File(projectPath + "/ksql-migrations.properties").createNewFile();
```

Comment:
The new config should have some default properties, and perhaps commented properties names in case the user wants to customize it.

Reply:
+1 but I was going to suggest this as a follow-up. I think we should update the `new` command to take in the ksqlDB server URL, which we can then pre-populate into the config file. This is the only required config in the config file, so we definitely want it.

I don't feel strongly about whether we populate some of the other configs with defaults or not. Could be nice for some of the ones we expect users are more likely to want to change.

Reply:
Agreed, but I'll leave this as a follow-up